modules/go-health-monitoring.adoc

```diff
@@ -0,0 +1,19 @@
+[id="health-information-resources_{context}"]
+= Checking health information
+
+.Prerequisites
+
+* {gitops-title} Operator installed from *OperatorHub*.
+* Argo CD applications are in sync.
+
+.Procedure
+
+* The Operator will install the backend service in `openshift-gitops` namespace. Click *Environments* under the *Developer* perspective. 
```

Comment:
in "the" namespace

Reply:
You should clarify which Operator and which backend service.

Reply:
Also, the first sentence is a description and not a step of the procedure. Also the module is lacking an introduction sentence. Move the descriptions of this procedure before the prerequisites to form an introduction sentence, and all will be according to our style guides.

Reply:
Since the procedure seems to list a sequence of steps, use a numbered list and not a bulleted list.